lua/wikis/commons/MatchTicker.lua

```diff
@@ -169,6 +171,10 @@ function MatchTicker:init(args)
 
 	config.hideTournament = Logic.readBool(args.hideTournament or Table.isNotEmpty(config.tournaments))
 
+	if config.displayGameIcons then
+		table.insert(config.queryColumns, 'game')
+	end
```

Comment:
Just always query it instead, the size diff is minimal in terms of percentage for this